x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/find_rules_status_route.ts

```diff
@@ -46,37 +49,59 @@ export const findRulesStatusesRoute = (router: IRouter) => {
         }
     */
```

Comment:
Lol, this comment and the way the `*/` is indented makes this file look a bit silly to me. Is this comment still valid on these lines btw?

```ts
// build return object with ids as keys and errors as values.
      /* looks like this
        {
            "someAlertId": [{"myerrorobject": "some error value"}, etc..],
            "anotherAlertId": ...
        }
    */
```

If not, can we remove it or move this to where maybe somewhere better? It might be better as just JSDocs around the route to not interrupt "the code reader" so to speak around line 18 above the REST route entry?

```ts
export const findRulesStatusesRoute = (router: IRouter) => {
```

Reply:
I will add JSDocs for the route 👍